### ISSUE TITLE
Training schedule and hyperparams configuration.

### ISSUE THREAD

**Issue by YJHMITWEB** [fizyr/keras-retinanet#515]
In your repo, the batch-size is 1 for 800 x 1333 images, so the learning_rate is quite small as 0.00001.
The question is, in focal loss paper, they set the batch_size=16 and learning_rate=0.01, so why I adjust the image size to be 240 x 400 hence the batch_size=16 on one Titan Xp, and I set the learning_rate=0.01 as well, however, the model cannot converge. Also, I tried a few times that it seems the model can only converge when learning_rate < 0.0001 no matter how large the batch_size is.
And also, why you use adam optimizer while in the paper they used sgd optimizer, and why did you set the clipnorm=0.001?
Thx!

**Comment by hgaiser** [comment]
Reducing image sizes to 240x400 will cause very small objects, which are more difficult to detect. I expect the quality to go down drastically when you reduce image sizes like that. They have bs=16 because they have a lot of GPUs and they split the load among them. We don't have that, so our default settings differ slightly from theirs.

We use adam because at the time we got good results using adam, but exploding gradients using SGD. I'm still open to SGD though, but I haven't tried it in a long time.
